### ISSUE TITLE
bug: Exception when download the report in latest version

### ISSUE THREAD

**Issue by highkay** [SecurityRiskAdvisors/VECTR#108]
When I click the download report button, a popup alert show "cancelled", and I got the following exception in the log.
```
tomcat_1     | Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "redTeamStatuses" (class com.sra.modelinterface.testcase.forms.TestCaseQueryFormImpl), not marked as ignorable (2 known properties: "excludes", "includes"])
tomcat_1     |  at [Source: (org.apache.cxf.transport.http.AbstractHTTPDestination$1); line: 1, column: 21] (through reference chain: com.sra.modelinterface.testcase.forms.TestCaseQueryFormImpl["redTeamStatuses"])
tomcat_1     |  at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:62)
tomcat_1     |  at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:834)
tomcat_1     |  at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:1093)
tomcat_1     |  at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1477)
tomcat_1     |  at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1455)
tomcat_1     |  at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:282)
tomcat_1     |  at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:140)
tomcat_1     |  at com.fasterxml.jackson.databind.ObjectReader._bind(ObjectReader.java:1583)
tomcat_1     |  at com.fasterxml.jackson.databind.ObjectReader.readValue(ObjectReader.java:964)
tomcat_1     |  at com.fasterxml.jackson.jaxrs.base.ProviderBase.readFrom(ProviderBase.java:814)
tomcat_1     |  at org.apache.cxf.jaxrs.utils.JAXRSUtils.readFromMessageBodyReader(JAXRSUtils.java:1406)
tomcat_1     |  at org.apache.cxf.jaxrs.utils.JAXRSUtils.readFromMessageBody(JAXRSUtils.java:1358)
tomcat_1     |  at org.apache.cxf.jaxrs.utils.JAXRSUtils.processRequestBodyParameter(JAXRSUtils.java:874)
tomcat_1     |  at org.apache.cxf.jaxrs.utils.JAXRSUtils.processParameters(JAXRSUtils.java:819)
tomcat_1     |  at org.apache.cxf.jaxrs.interceptor.JAXRSInInterceptor.processRequest(JAXRSInInterceptor.java:214)
tomcat_1     |  at org.apache.cxf.jaxrs.interceptor.JAXRSInInterceptor.handleMessage(JAXRSInInterceptor.java:78)
tomcat_1     |  ... 42 more
```

**Comment by SRAPSpencer** [comment]
Thanks for reporting this! We're working on a hotfix release to resolve this. Should be out tomorrow or next week.

**Comment by carlvonderheid** [comment]
Should be fixed in 6.1.1, thanks for bringing this to our attention.
